data/mods/Magiclysm/Spells/technomancer.json

```diff
@@ -344,6 +344,32 @@
     "max_range": 30,
     "range_increment": 1
   },
+  {
+    "id": "summon_floating_disk",
+    "type": "SPELL",
+    "name": "Summon floating disk",
+    "description": "Summons a floating disk that is sworn to carry your burdens.",
+    "valid_targets": [ "ground" ],
+    "flags": [ "SOMATIC", "VERBAL", "CONCENTRATE" ],
+    "effect": "summon_vehicle",
+    "spell_class": "TECHNOMANCER",
+    "energy_source": "MANA",
+    "vehicle_id": "floating_disk",
+    "difficulty": 3,
+    "max_level": 20,
+    "base_casting_time": 100,
+    "casting_time_increment": -1,
+    "final_casting_time": 80,
+    "base_energy_cost": 150,
+    "energy_increment": -2,
+    "final_energy_cost": 100,
+    "min_range": 1,
+    "max_range": 10,
+    "range_increment": 1,
+    "min_duration": 32000,
```

Comment:
these durations should be measured in hours, not minutes. maybe 1 hour up to 24 hours.